### DISCUSSION TITLE
Validation and Parachain

### DISCUSSION THREAD

**Discussion by Crypte** [capsule-corp-ternoa/ternoa-hub#15] [General]
Hello everyone, as you know Ternoa is intended to be a parachain (phase 5 of the mainnet) and being myself the owner of 3 active validators (CRYPTENODE 01, 02 and 03), I wonder how it will happen. 

A parachain implies a complete change of the validation system, no more need of specific validator because using the validators of the Polkadot relay chain. 

But we will need collators, how will the transition be done? Will everyone be able to launch one? Is this the launch of a new blockchain? 

I know this is a long way off but I think this is essential information for current validators
I've already briefly discussed this with Marko but I want this to be a discussion that everyone can have

 Take care

**Comment by peshwar9** [comment]
Hello @Gitcrypte ,
That's right, move to parachain would be like starting a new chain with a genesis block, but there would be two sets of operators involved in block production and finalisation - the collators of the parachain, and the validators of Polkadot/Kusama. You can find more details of role of collators here: https://wiki.polkadot.network/docs/learn-collator.

In terms of how the transition will be done, we will be able to share details once the planning is done (we have completed phase 2 of Mainnet in July'22, and working on phase 3).
